api/src/templates/new_member.body.html

```diff
@@ -1,167 +1,194 @@
+<p>As a new member, you probably have a lot of questions. In this email we provide you with a summary of what you heard on the introduction course and a few resources to get you started.</p>
 
-<p>As a new member, you probably have a lot of questions about the Stockholm Makerspace. In this email you'll find the info you need to answer most, if not all, of your questions, so please read this email carefully!
-</p>
-
-<h3>Member info</h3>
+<h3>Your member information:</h3>
 <p>Name: {{member.firstname}} {{member.lastname}}<br/>
     Email: {{member.email}}<br/>
     Membership number: {{member.member_number}}<br/>
 </p>
 
-<p>We’ll use this email if we need to reach you. Therefore it’s important that you ensure that it works, or notify us regarding new email info. Your membership number is used for our database, and you’ll need it to sign up for courses and workshops.</p>
-
-<h3>TWO TYPES OF MEMBERSHIP</h3>
-<p><b>Base Membership</b><br/>
-    As a member in the organisation you’re welcome to attend our events and workshops, as well as vote at the annual meeting. If you wish to do independent projects on your own on the premises, you need to become a makerspace access member.</p>
-
-<p><b>Makerspace Access</b><br/>
-    The premises are open 24/7 for members who have Makerspace Access, all year long, but to start gaining access you need to attend a new member introduction. New member introductions occur approximately every two weeks, and they’re called "medlemsintroduktion" in our <a href="https://makerspace.se/kalendarium">calendar</a>. To get access you need to sign an agreement where you agree to adhere to <a href="https://wiki.makerspace.se/Rules_for_Makerspace"></a>our rules</a>, and you also need to show an ID.</p>
-
-<h3>I want to go to a workshop! When is the next workshop?</h3>
-<p>All activities are run by volunteers, like you and me. We help each other learn the machines so that everyone can work independently. Share what you can, and ask for help if you get stuck. Some machines require that you have gone to a course for in order to be able to use, for example:  welding (svetsrummet), laser cutter, embroidery-machine and resin printer. Courses are advertised in the Facebook group and on Slack and can be found on the calendar. There is no queue - first come, first served.</p>
-
-<h3>I'm really good at something. Can I lead a workshop?</h3>
-<p>Absolutely! Email the board at <a href="mailto:info@makerspace.se">info@makerspace.se</a> to talk about the set up, creating a sign-up, and posting the course in the calendar. If you're unsure there is interest for your specific area of knowledge you can ask in the Facebook-group or on Slack.</p>
+<p>We will use this email address to reach out to you. If it is not right, please email <a href="mailto:info@makerspace.se">info@makerspace.se</a>.</p>
```

Comment:
Se kommentar på svenska delen först. Men annars så är det mer korrekt att skriva "if this is not correct".

Reply:
Fixing it. thank you